RELEASE-NOTES.md

```diff
@@ -2,9 +2,8 @@
 
 ## Release 2.13.6
 
-### New default option for results storage
-
-
+### Specify a default result storage block as a setting
+Previously, specifying result storage blocks necessitated changes in the @flow / @task decorator. Now, the `PREFECT_DEFAULT_RESULT_STORAGE_BLOCK` setting allows users to set a default storage block on a work pool or via infra overrides for a deployment. This enhancement enables easier swapping of result storages by just updating the environment in the UI or in your prefect.yaml, eliminating the need to alter your flow source code.
```

Comment:
Can you add an example of how you would set this value for a work pool or deployment?